modules/connectors/ninja_van/karrio/providers/ninja_van/rate.py

```diff
@@ -0,0 +1,58 @@
+
+import typing
+import karrio.lib as lib
+import karrio.core.units as units
+import karrio.core.models as models
+import karrio.providers.ninja_van.error as error
+import karrio.providers.ninja_van.utils as provider_utils
+import karrio.providers.ninja_van.units as provider_units
+
+
+def parse_rate_response(
+    _response: lib.Deserializable[dict],
+    settings: provider_utils.Settings,
+) -> typing.Tuple[typing.List[models.RateDetails], typing.List[models.Message]]:
+    response = _response.deserialize()
+
+    messages = error.parse_error_response(response, settings)
+    rates = [_extract_details(rate, settings) for rate in response]
+
+    return rates, messages
+
+
+def _extract_details(
+    data: dict,
+    settings: provider_utils.Settings,
+) -> models.RateDetails:
+    rate = None  # parse carrier rate type
+
+    return models.RateDetails(
+        carrier_id=settings.carrier_id,
+        carrier_name=settings.carrier_name,
+        service="",  # extract service from rate
+        total_charge=lib.to_money(0.0),  # extract the rate total rate cost
+        currency="",  # extract the rate pricing currency
+        transit_days=0,  # extract the rate transit days
+        meta=dict(
+            service_name="",  # extract the rate service human readable name
+        ),
+    )
```

Comment:
This needs to be implemented. Since the Tariff API returns 

```
{
  "data": {
    "total_fee": 90000
  }
}
```

you need to collect the data and forward to the Karrio RateDetails